property_tests/tests/class_syntax/blob/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "scripts": {
+        "test": "tsx test/test.ts"
+    },
+    "dependencies": {
+        "azle": "0.23.0"
+    },
+    "devDependencies": {
+        "tsx": "^4.15.7",
+        "typescript": "^5.2.2"
```

Comment:
Can you get rid of typescript and see if that works? If it works, can you remove it from all property tests?

Reply:
done